### ISSUE TITLE
Introspection fails, report 1296: Unable to resolve scalar field

### ISSUE THREAD

**Issue by Simskii** [prisma/prisma2#1405]
Fails to introspect Prisma1 ManyToMany tables.
Report id 1296

```
model ServiceToExtraServices {
  A Service @relation("ServiceTo_ServiceToExtraServices_A")
  B Service @relation("ServiceTo_ServiceToExtraServices_B")

  @@index([B], name: "B")
  @@map("_ServiceToExtraServices")
  @@unique([A, B], name: "ServiceToExtraServices_AB_unique")
}
```

```
Schema parsing
thread 'main' panicked at 'Unable to resolve scalar field 'A'', src/libcore/option.rs:1185:5
stack backtrace:
   0: <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt
   1: core::fmt::write
```

**Comment by janpio** [comment]
Possibly tracked in https://github.com/prisma/prisma2/issues/1377

**Comment by pantharshit00** [comment]
Confirmed via bug report.

**Comment by janpio** [comment]
This should be fixed with the current `alpha` @Simskii! (The fix will also be included in the upcoming preview release, `2.0.0-preview022`)
